### DISCUSSION TITLE
Which specific Lean version will leandojo use?

### DISCUSSION THREAD

**Discussion by wclsn** [lean-dojo/LeanDojo#148] [Q&A] [answered]
Hello, everyone
I am curious about the specific lean version that Dojo will utilize when I use it to interact with Lean, such as running run_tac(). I have multiple toolchains installed on my computer, and I would like to know which particular lean version Dojo will employ for these interactions, the default one?
![image](https://github.com/lean-dojo/LeanDojo/assets/84527545/1e84c457-437c-44a3-818d-8ed284c20140)

**Comment by darabos** [comment, marked as answer]
I think you get to `run_tac()` from picking a theorem from a repo. That repo has a `lean-toolchain` file that specifies the Lean version. LeanDojo will use that version.
